### ISSUE TITLE
Taxonomic name resolving to wrong page

### ISSUE THREAD

**Issue by Archilegt** [NaturalHistoryMuseum/scratchpads2#6399]
**Describe the bug**
The page https://myriatrix.myspecies.info/myriatrix/newportia-mexicana
is resolving to https://myriatrix.myspecies.info/myriatrix/newportia-scolopendrides-mexicana

Could you please fix this?

**Comment by Archilegt** [comment]
Could someone please take a look at this? It may be possible to solve it in the same way than Indexing error for a taxon page #6280

**Comment by Archilegt** [comment]
This _Scolopocryptops_ issue is solved, so I will close it. The _Newportia_ issue still needs to be solved.

**Comment by Archilegt** [comment]
Closed by mistake. This is the _Newportia_ issue, which is not solved.

**Comment by Archilegt** [comment]
I see. The URL redirect was set on the term https://myriatrix.myspecies.info/myriatrix/newportia-scolopendrides-mexicana instead of on the term https://myriatrix.myspecies.info/myriatrix/newportia-mexicana. That the cause of a name resolving to another page is not set on the source name editing interface but on the target name interface seems counterintuitive to me. That is why I was not finding the cause. I deleted the URL redirect and the problem is now solved.
